src/actions/index.js

```diff
@@ -278,9 +208,11 @@ export function updateInterestSubscription(interest, subscribed) {
   var NotificationSettings = NativeModules.NotificationSettings
   return () => {
     return new Promise((resolve) => {
-      NotificationSettings.setInterestSubscription(interest, subscribed).then((message) => {
-        return resolve(message)
-      })
+      if (subscribed !== undefined) {
```

Comment:
Cool. I think this will take care of one of the errors I've been experiencing in testing.